.travis.yml

```diff
@@ -5,8 +5,9 @@ jdk:
 
 android:
   components:
-    - android-22
-    - build-tools-22.0.1
+    - tools
```

Comment:
There is an issue in travis cause of which it was saying that i cant not find build-tools-23.0.2. I found on net and found that it is gone if you add tools, So I don't know much at present except of travis fail to find build-tools-23.0.2 without it.